e2e/interchaintestv8/sp1_ics07_test.go

```diff
@@ -53,7 +54,7 @@ type SP1ICS07TendermintTestSuite struct {
 
 // SetupSuite calls the underlying SP1ICS07TendermintTestSuite's SetupSuite method
 // and deploys the SP1ICS07Tendermint contract
-func (s *SP1ICS07TendermintTestSuite) SetupSuite(ctx context.Context) {
+func (s *SP1ICS07TendermintTestSuite) SetupSuite(ctx context.Context, opArgs ...string) {
```

Comment:
I'm not too fond of this approach. The test setup is not very easy to understand and we are leaking the operator parameters out to all the tests. I think it would be better if we just took in a prover enum or something and converted that to the correct arguments when needed.